src/common/MovieTile/styled.js

```diff
@@ -0,0 +1,87 @@
+import { styled } from "styled-components";
+
+export const Wrapper = styled.div`
+  width: 324px;
+  height: 650px;
+  border-radius: 5px;
+  box-shadow: 0px 4px 12px 0px rgba(186, 199, 213, 0.5);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  gap: 8px;
+`;
+
+export const Poster = styled.img`
+  display: block;
+  width: 292px;
+  height: 434px;
+  margin-top: 16px;
+  border-radius: 5px;
+`;
+
+export const Description = styled.div`
+  width: 292px;
+`;
+
+export const Title = styled.div`
+  width: 292px;
+  font-size: 22px;
+  font-weight: 500;
+  line-height: 1.3;
+  text-align: left;
+  margin: 8px 0;
+`;
+
+export const Year = styled.div`
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.5;
+  color: grey;
+`;
+
+export const Tags = styled.div`
+  width: 279px;
+  height: 36px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 8px;
+  margin: 8px 0;
+`;
+
+export const Tag = styled.div`
+  height: 36px;
+  padding: 8px 16px 8px 16px;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 20px;
+  text-align: left;
+  background-color: #E4E6F0;
+`;
+
+export const Rating = styled.div`
```

Comment:
you can add a color there, to match the one from figma (#7E839A in hex)

Reply:
This color (#7E839A in hex) is too dark in my display. I stayed with the current color.